src/main/resources/data/computercraft/lua/rom/apis/rednet.lua

```diff
@@ -394,8 +394,8 @@ function run()
                 if type(tMessage) == "table" and type(tMessage.nMessageID) == "number"
                     and tMessage.nMessageID == tMessage.nMessageID and not tReceivedMessages[tMessage.nMessageID]
                 then
-                    tReceivedMessages[tMessage.nMessageID] = true
-                    tReceivedMessageTimeouts[os.startTimer(30)] = tMessage.nMessageID
+                    tReceivedMessages[tMessage.nMessageID] = os.epoch("utc")
```

Comment:
I think it's more sensible to use `os.clock()` here instead. It's tied to TPS (like `os.startTimer`) and guaranteed to be monotonic, which `os.epoch` isn't. Otherwise we're going to end up with some odd behaviour in the case of daylight savings time!

I think what it might be more intuitive to do here is set the deadline here instead, so:

```suggestion
                    tReceivedMessages[tMessage.nMessageID] = os.clock() + 9.5
```